DI-Policies.md

```diff
@@ -0,0 +1,51 @@
+# Statement
+
+Historically, the wedding industry has been massively lacking in areas of representation and equity, especially across race, gender, and sexual orientation. The fact that interracial marriage wasn’t fully legal in the U.S. until 1967, and same sex marriage wasn’t legal until 2015, has been reflected in wedding industry practices and imagery since its beginning. And to this day, it’s painfully clear just how racist, sexist, homophobic, and generally prejudiced the industry still is.
+
+It’s up to wedding vendors and industry leaders to correct this. We have a responsibility to create spaces and content where all people feel seen, safe, and valued. 
+
+It’s important to me as a wedding business owner to not only implement policies that address these issues, but also hold myself accountable to those policies. So, I’m sharing this document publicly as one part of my own efforts to do better and be better. This not an “arrival”, but rather part of an ongoing process that will evolve and change over the lifetime of my business as I learn and iterate to create more inclusive, welcoming, and safe spaces for all folx.
+
+# Specific policies
+
+These are the specific actions I commit to taking as integral parts of how I do business moving forward. 
+
++ Listening + amplifying
+
+   + Listen to and amplify the stories and experiences of marginalized groups in the industry, and out of it.
+
+   + Speak up when witness to problematic statements, situations, and practices.
+
+   + Own my own mistakes when - not if - I make them, make reparations, and do better next time.
+
++ Regular audits of website and social media feeds
+
+   + Quarterly for website.
+
+   + Monthly for social.
+
++ Vetting subcontractors + partners
+
+   + Interview second shooters, editors, and any others I work with or hire to ensure inclusive practices and the comfort/safety of my clients.
+
++ Continuing education 
+
+   + Minimum two PAID workshops a year (pay Black folx and other marginalized individuals for their work).
+
+   + Before participation in any styled shoots, workshops, or other events, inquire into their own D+I policies. 
+
++ Networking + referrals
+
+   + Regularly and consistently seek out, work with, and refer underrepresented vendors.
```

Comment:
Again, too vague?

Reply:
For now, I think it's good enough, maybe next year, when you review this policy, you can add a number next to it. I am a sucker for a good KPI.

Reply:
I think it would be great to include a specific goal here. Something like 'aim to work with underrepresented vendors for at least 50% of my shoots" or "1 in 3 referrals will be for vendors of underrepresented groups"

Even if you don't hit it, it's a starting point and something tangible to work towards.

Reply:
You both make good points here. I'll see what I can come up with as far as starting numbers, but I agree with Pam that I'll probably have a better sense of what's feasible, and what I should be reaching for, after a year or so.